cg/meta/archive/archive.py

```diff
@@ -188,7 +186,108 @@ def add_samples_to_files(self, files_to_archive: list[File]) -> list[FileAndSamp
         adds it to the list which is returned."""
         files_and_samples: list[FileAndSample] = []
         for file in files_to_archive:
-            sample: Optional[Sample] = self.get_sample(file)
-            if sample:
+            if sample := self.get_sample(file):
                 files_and_samples.append(FileAndSample(file=file, sample=sample))
         return files_and_samples
+
+    def update_status_for_ongoing_tasks(self) -> None:
+        """Updates any ongoing jobs with a finished timestamp, should they be completed."""
```

Comment:
```suggestion
        """Updates any ongoing jobs with a finished timestamp."""
```

Reply:
But it doesn't update any ongoing job? Only if it has finished. But maybe that is implied.

Reply:
Can it be ongoing and completed at the same time? That is confusing

Reply:
No, but we do not know if it is finished yet, so for any jobs we have marked as ongoing, we ping ddn and check. If they say that the job is finished, we update it on our side

Reply:
I do find `should they be completed` to be a bit confusing. Is `Updates any completed jobs with a timestamp` a good alternative?

Reply:
Sure - updated it!